starter-code/server.js

```diff
@@ -172,7 +175,7 @@ function loadArticles() {
 
 function loadDB() {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Put your response here: This corresponds to #5, response. This is the 'loadArticles' function. This is the 'C' part of CRUD if the table does not exist. If it does exist then it will be the 'U' part of CRUD, or update.
```

Comment:
This can be considered the C in CRUD, but check diagram and where loadDB is being called